packages/vsce/package.json

```diff
@@ -763,7 +801,8 @@
             "URIMap": true,
             "WebService": true,
             "JVMServer": true,
-            "Bundle": true
+            "Bundle": true,
+            "JVMEndpoint": true
```

Comment:
I don't think we need this. This list is to select which trees are shown under a region. JVM Endpoints are only under JVM servers so will show if JVM servers are present. Similar to BundlePart behaviour